api/src/services/opportunities_v1/search_opportunities.py

```diff
@@ -148,7 +153,10 @@ def _get_search_request(params: SearchOpportunityParams) -> dict:
 
     # Query
     if params.query:
-        builder.simple_query(params.query, SEARCH_FIELDS)
+        filter_rule = FILTER_RULE_MAPPING.get(
```

Comment:
I could alternatively access using `FILTER_RULE_MAPPING[params.experimental.scoring_rule]` since we already handling the default case